nile-backend/src/main/java/com/tp/Nile/models/Role.java

```diff
@@ -25,6 +27,12 @@
     @Column(length = 60)
     private RoleName name;
 
+    @ManyToMany(fetch = FetchType.LAZY)
```

Comment:
I'm not sure if the many to many is necessary here because it already exists on the User model.